monkey/infection_monkey/network_scanning/tcp_scanner.py

```diff
@@ -16,7 +16,7 @@
 
 
 def scan_tcp_ports(
-    host: str, ports_to_scan: Iterable[int], timeout: float
+    host: str, ports_to_scan: Collection[int], timeout: float
```

Comment:
Why use `Collection` over `Iterable`?

Reply:
`Collection` seems to be sized iterable. If the `len()` is used somewhere down the line, then it would make sense